src/main/java/com/notification/controller/NotificationController.java

```diff
@@ -75,4 +76,12 @@ public ResponseEntity<ResponseModel<InvitationStatusResponse>> updateStatus(
         log.info("Endpoint: /put update status ends with update status: {}", response.getStatus());
         return ResponseEntity.ok(response);
     }
+
+    @PostMapping(value = "/forgot-password")
+    public ResponseEntity<ResponseModel<String>> sendPasswordResetLinkEmailToUser(@RequestBody PasswordResetTokenRequest passwordResetTokenRequest){
```

Comment:
Instead of using user class object inside passwordResetTokenRequest, use fullName and sentToEmail.

Reply:
Resolved